scout/server/blueprints/variants/templates/variants/components.html

```diff
@@ -21,6 +21,25 @@
   </div>
 {% endmacro %}
 
+{% macro svs_funct_anno_cell(variant) %}
+  <div class="d-flex justify-content-between align-items-center">
+    <div>
+      {% if variant.functional_annotations|length >= 5 %}
+        <a class="mr-3" data-bs-toggle="collapse" href="#_functanno_{{variant._id}}" role="button" aria-expanded="false" aria-controls="_{{variant._id}}">{% if variant.missing_data %}[first 30 genes]{% else %}[...]{% endif %}</a>
```

Comment:
Yes, we'll just have to live with that; otw this becomes a rather large project. It is now understood, and says so clearly much like on the variant page.

Reply:
What we could improve is parsing the large SVs not to include the first 30something genes, but genes of interest

Reply:
Yes, definitely - like use OMIM, clinical list, near-ends of balanced events... But better make a nice common routine for that separately, right?!

Reply:
yes!